src/sentry/migrations/0520_migrate_worldmap_widgets.py

```diff
@@ -0,0 +1,66 @@
+# Generated by Django 3.2.20 on 2023-07-24 21:37
+
+from django.db import migrations
+
+from sentry.models import DashboardWidgetQuery
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+def migrate_worldmap_widgets_to_table_widgets(apps, schema_editor):
+    # World map visulization has display_type 5
+    widgetQueries = DashboardWidgetQuery.objects.select_related("widget").filter(
+        widget__display_type=5
+    )
```

Comment:
You're around the 10k row count. You *could* hit a migration timeout when updating rows. If you wanted to be totally safe an `is_dangerous = True` migration would safer albeit much slower.

Reply:
Yeah I'd rather be safe. How much longer would it take @markstory?